src/components/Button/Button.tsx

```diff
@@ -24,6 +25,10 @@ export interface Props {
    * Toggles button that fills the full width of its container
    */
   fullWidth?: boolean;
+  /**
+   * Visually hides button text for icon-only buttons for accessibility
+   */
+  hideText?: boolean;
```

Comment:
We explicitly removed `hideText` in https://github.com/chanzuckerberg/edu-design-system/pull/889 because you can use `aria-label` instead of `text` to get the same effect.